<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAActionFTAKinematicMultiOutputWidget</class>
 <widget class="QWidget" name="GTAActionFTAKinematicMultiOutputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Name              </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NameLE"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Reference File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RefFilePathLE">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BrowseRefFilePB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/images/forms/img/OpenFolder.png</normaloff>:/images/forms/img/OpenFolder.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Variables</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Input1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="InputParam1LE"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Input2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="InputParam2LE"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Input3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="InputParam3LE"/>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="input2SearchPB">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QToolButton" name="input3SearchPB">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="input1SearchPB">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Monitorings</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Output1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="OutputParam1LE"/>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="output1SearchPB">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Output2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="OutputParam2LE"/>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="output2SearchPB">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Output3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="OutputParam3LE"/>
           </item>
           <item row="2" column="2">
            <widget class="QToolButton" name="output3SearchPB">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Output4</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="OutputParam4LE"/>
           </item>
           <item row="3" column="2">
            <widget class="QToolButton" name="output4SearchPB">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Output5</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="OutputParam5LE"/>
           </item>
           <item row="4" column="2">
            <widget class="QToolButton" name="output5SearchPB">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../resource.qrc">
               <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>7</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Test Parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="Epsilon1Lbl">
             <property name="text">
              <string>Epsilon_1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="Epsilon1LE"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="Epsilon2Lbl">
             <property name="text">
              <string>Epsilon_2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="Epsilon2LE"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="WaitPilOrderLbl">
             <property name="text">
              <string>wait_pil_order</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="WaitPilOrderLE"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="NumLinesLbl">
             <property name="text">
              <string>NumOfLines</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="NumLinesLE"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>35</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>frame</zorder>
     <zorder>frame_2</zorder>
     <zorder>frame4</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
